python/cugraph/snmg/link_analysis/mg_pagerank_wrapper.pyx

```diff
@@ -27,7 +27,10 @@ from librmm_cffi import librmm as rmm
 import numpy as np
 
 
-def mg_pagerank(src_ptrs_info, dest_ptrs_info):
+def mg_pagerank(src_ptrs_info,
+                dest_ptrs_info,
+                alpha=0.85,
+                max_iter=30,):
```

Comment:
the default of max_iter is 30 and dask_cugraph is 50. Could be a confusion as to the default value.

Reply:
Good catched. Fixed in dask-cugraph https://github.com/rapidsai/dask-cugraph/commit/91fbdf4eef2448f264c33c6c0b5b0e2e7f4bae0e